pages/index.tsx

```diff
@@ -78,8 +108,8 @@ const Home: NextPage<{ initialGrid: Color[][] }> = ({ initialGrid }) => {
         grid={<Grid onCellSelect={handleCellSelect} grid={grid} />}
         actions={
           <>
-            <button onClick={() => copyHashToClipboard(grid)}>save</button>
-            <button onClick={() => resetGrid()}>reset</button>
+            <button onClick={() => saveClickActions()}>save</button>
+            <button onClick={() => resetClickActions()}>reset</button>
```

Comment:
There's often a convention to call these event handlers "handleResetClick" or "handleSaveClick" (which you don't have to follow but just thought I'd mention it). Also you don't need to wrap this in an arrow function, you can just pass the function directly to onClick without calling it